Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -329,13 +329,17 @@ public void OnSessionControllerConnected(SessionInfo session)
         }
 
         /// <inheritdoc />
-        public void CloseIfNeeded(SessionInfo session)
+        public async Task CloseIfNeededAsync(SessionInfo session)
         {
             if (!session.SessionControllers.Any(i => i.IsSessionActive))
             {
                 var key = GetSessionKey(session.Client, session.DeviceId);
 
                 _activeConnections.TryRemove(key, out _);
+                if (!string.IsNullOrEmpty(session.PlayState?.LiveStreamId))
+                {
+                    await _mediaSourceManager.CloseLiveStream(session.PlayState?.LiveStreamId).ConfigureAwait(false);
```

Comment:
I don't remember if `.Value` is required, but the `?` is unnecessary
```suggestion
                    await _mediaSourceManager.CloseLiveStream(session.PlayState.LiveStreamId).ConfigureAwait(false);
```